### ISSUE TITLE
Show correct URL of local server (e.g. in WSL)

### ISSUE THREAD

**Issue by leonexcc** [symfony-cli/symfony-cli#82]
In WSL with Ubuntu the IP of the local server is not 127.0.0.1 but [::1]. If you open the http://127.0.0.1:8000 link in the terminal it will not work. What is working is http://[::1]:8000 or more universaly http://localhost:8000. (see also the old issue at https://github.com/symfony/cli/issues/418)

**Comment by huiman** [comment]
with this problem, I has found that profiler is also missing.

**Comment by henrikac** [comment]
I believe this was fixed earlier today by your PR, @fabpot ? :)

**Comment by tucksaun** [comment]
Not necessarily: it will depend if the user manually attached a domain and uses the proxy or not.
